src/index.ts

```diff
@@ -61,17 +58,16 @@ export async function initializeFromJsonFile(jsonFilepath: string, options?: IOp
  * @param aiKey Key of Application Insights.
  * @param options debug: if set as true, debug information be printed to console.
  */
-export function initialize(extensionId: string, version: string, aiKey: string | string[], options?: IOptions): void {
+export function initialize(extensionId: string, aiKey: string | string[], options?: IOptions): void {
```

Comment:
Keep the API as previous so that the downstream only need bump version in package.json.